dev/README.md

```diff
@@ -171,10 +171,8 @@ The local infra setup creates pods for:
 To build and run the infra setup command do:
 
 ```
-pushd e2e/testenv/infra
-  go build -o . ./...
-popd
-./e2e/testenv/infra/infra setup
+go build -o infra ./e2e/testenv/infra/
+./infra setup
```

Comment:
This would become inconsistent with `dev/setup-{single,multi}-cluster`, so we should either edit them too or keep this as is.

Reply:
Thanks, I didn't know those (and missed them in my search)